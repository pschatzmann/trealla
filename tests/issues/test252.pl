:-initialization(main).

main :-
	writeq(Ö-Œ = s-t), nl,
	V = -'Ö'-'Œ'+(.)+'A',
	writeq(V), nl.
