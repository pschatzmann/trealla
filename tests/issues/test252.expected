_2-_3= s-t
_2-_3= s-t
-'Ö'-'Œ'+'.'+'A'
